projects/components/src/link/link.component.ts

```diff
@@ -12,6 +12,8 @@ import { EMPTY, Observable } from 'rxjs';
     <a
       *htLetAsync="this.navData$ as navData"
       class="ht-link"
+      target="{{ this.openInNewTab ? '_blank' : '' }}"
```

Comment:
how's this differ from the existing way of doing this?

This component takes an ExternalNavigationParams as an input, which includes window handling.

Reply:
Hmm. I thought it suspicious that we didn't have this already. I though that might be how, but it didn't seem to work when I was playing with it. I'll circle back and try again.

Reply:
Yep.

Reply:
ExternalNavigationParams option should support this.